### PR TITLE
Shut down modal

### PR DESCRIPTION
**Pull request by aman1376**
## Changes Proposed

Created a popup when the shut down button is clicked.

Screenshot (56)
Screenshot (57)

## Check List

- [X] The title of my pull request is a short description of the changes
- [X] This PR relates to some issue: 
- [X] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

**Comment by aman1376** [comment]
@chad11111 please have a look.

File: dashboard/src/layout/Header.tsx [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+  const [signOut,setSignOut]=useState(false);
+
+  const handleClick=()=>{
+    setSignOut(true);
+  }
+
   return (
     <header className="app-header">
+    {
+          signOut && <SignOutpopup/>
```
Comment (chad11111) [inline dashboard/src/layout/Header.tsx:16]: please use the popup component.

File: dashboard/package.json [hunk 2]
```diff
@@ -4,9 +4,13 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "@emotion/react": "^11.10.6",
```
Comment (chad11111) [inline dashboard/package.json:7]: why do we need this?

File: dashboard/package.json [hunk 3]
```diff
@@ -4,9 +4,13 @@
...
+    "@emotion/styled": "^11.10.6",
```
Comment (chad11111) [inline dashboard/package.json:8]: why do we need this?

File: dashboard/package.json [hunk 4]
```diff
@@ -4,9 +4,13 @@
...
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.13",
```
Comment (chad11111) [inline dashboard/package.json:10]: do we need this ? cant we just use icons ?

**Review by chad11111 — APPROVED** [review]

File: dashboard/src/layout/Header.tsx (hunk 1 above)
Comment (aman1376) [reply dashboard/src/layout/Header.tsx:16]: @chad11111 for that i would need to install the reactjs-popup dependency do you want me to do this way?

File: dashboard/package.json (hunk 4 above)
Comment (aman1376) [reply dashboard/package.json:10]: @chad11111 i'll try to check with icons i am not sure about that. i am comfortable with material-ui and its a  bit more styled but if you want me to do with icons i'll look into it.

File: dashboard/src/layout/Header.tsx (hunk 1 above)
Comment (chad11111) [reply dashboard/src/layout/Header.tsx:16]: No, don't install anything we wrote our own modal.

Have a look here.
https://github.com/komodorio/helm-dashboard/tree/helm-dashboard-v2/dashboard/src/components/modal

File: dashboard/package.json (hunk 4 above)
Comment (chad11111) [reply dashboard/package.json:10]: We dont want to use mui.
Please only use the icons.

File: dashboard/package.json (hunk 3 above)
Comment (chad11111) [reply dashboard/package.json:8]: can u use this https://react-icons.github.io/react-icons/?

**Comment by aman1376** [comment]
@chad11111 pls have a look

**Pull request merged**